### ISSUE TITLE
Add autofocus on tags input

### ISSUE THREAD

**Issue by avdieiev** [Pocket/extension-save-to-pocket#47]
## Expected Behavior
Pocket tags input should be focused, when saving 
## Actual Behavior
Pocket tags input isn't focused
## Steps to Reproduce the Problem

1. Open page
2. Click "Save to pocket"

## Specifications

* Version: 3.0.3.0
* Browser: Chrome 65
* Operating System: Windows 10, MacOS High Sierra

**Comment by collectedmind** [comment]
👍 Thanks for pointing this out @avdieiev 

I implemented this fix and tested it locally. This will be included in the next release.

**Comment by collectedmind** [comment]
#50 - We need to finalize copy and then update all locales.

**Comment by feargswalsh92** [comment]
Does this mean that the cursor would automatically move to the tags box when you either hit the pocket button or your keyboard shortcut? If so yes this would be a great addition, if not I would like to create a new issue and have a crack at it.

**Comment by collectedmind** [comment]
@feargswalsh92 Yes, that is the intent. This functionality is complete and ready to add.  It will live behind a feature flag under options.

Love that you want to contribute!  Always open to suggestions and PR's

We're gonna try and lay out a road map of new features prior to releases and get a better release cadence worked out soon.  This should give a clearer picture of good places to jump in.

**Comment by feargswalsh92** [comment]
Awesome! In the meantime I'll trawl the code base for typos  😄 . I don't know for sure but I feel like even if the feature is turned on it should still require hitting the tab button to activate it to focus on the tabs box. I'm imagining a very particular use case where the user is already typing, wants to pocket the page they're typing on and doesn't in that instance want to add tags. Especially if it means that they need to move the cursor back to the text box they're typing on. As this recreates the problem that this feature is trying to solve in reverse.

**Comment by adsk2050** [comment]
Hey @nelsonomuto ! Is this feature implemented? If not, I am interested in taking up its development. Would love to have a chat today.

**Comment by dpenacton** [comment]
Hi, any updated on this feature? with the extension shortcut, this is the only feature missing in my workflow.

**Comment by ljubadr** [comment]
Any update on this?
